org.p3x.onenote.yml

```diff
@@ -0,0 +1,50 @@
+id: org.p3x.onenote
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    XDG_CACHE_HOME: /run/build/p3x-onenote/flatpak-node/cache
+    npm_config_cache: /run/build/p3x-onenote/flatpak-node/npm-cache
+    npm_config_nodedir: /usr/lib/sdk/node18
+    npm_config_offline: 'true'
+
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+
+name: p3x-onenote
```

Comment:
I doubt you managed to build with the manifest, even though you checked `yes` to that question

Reply:
The flatpak-builder warning above should be error. You shouldn't ignore them.

Reply:
you copied it wrong. I can tell you. The linter agrees with me, and that's because it's called before flaptak-builder even attemps.